prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -34,9 +31,6 @@
 # condition-assertedDate
 - id: "condition-assertedDate"
   url: "http://hl7.org/fhir/StructureDefinition/condition-assertedDate"
-# processing ID
-- id: "source-processing-id"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/source-processing-id"
```

Comment:
Same question as above

Reply:
It is no longer needed. That data is now in some resource property.